docs/core/config/mailbox/formats/imapc.md

```diff
@@ -3,6 +3,9 @@ layout: doc
 title: imapc
 dovecotlinks:
   imapc: imapc
+  imapc_quota:
+    hash: quota
+    text: imapc quota
```

Comment:
This text is used as the link text, so it should be titlecased. (I personally would write `"Imapc: Quota"` (you need to quote a title containing colons), but `Imapc Quota` is fine as well.)

Reply:
I think there are lots of existing link texts that aren't titlecased. But changed these.